backend/EasySpeak.Core/EasySpeak.Core.Common/DTO/UploadFile/NewFileDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace EasySpeak.Core.Common.DTO.UploadFile
+{
+    public class NewFileDto
+    {
+        public Stream Stream { get; set; } = null!;
+        public string FolderPath { get; set; } = null!;
```

Comment:
Won't be here, it's up to you set up folders

Reply:
Generally we should have the same folder(blob container) for all profile pictures